### PR TITLE
feat: update for Intellij 2023.1

### PR DESCRIPTION
**Pull request by nexoscp**
Update for Intellij 2023.1

### CODE REVIEW

**Comment by Pakisan** [comment]
@nexoscp hi!

I didn't merge [release/2.0.0](https://github.com/asyncapi/jasyncapi-idea-plugin/tree/release/2.0.0) into `master`, that's why you have conflicts. Please re-target your mr to branch, which I created for you - `release/2.1.0`

File: build.gradle.kts [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
-    id("org.jetbrains.intellij") version "1.12.0"
+    id("org.jetbrains.intellij") version "1.13.3"
     java
-    kotlin("jvm") version "1.8.0"
+    kotlin("jvm") version "1.8.10"
 }
 
 group = "com.asyncapi.plugin.idea"
-version = "2.0.0+jre17"
+version = "2.0.1+jre17"
```
Comment (Pakisan) [inline build.gradle.kts:8]: It's new feature, not bug fix. Please change to `2.1.0`

File: build.gradle.kts [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-    version.set("2022.3.1")
+    version.set("2023.1")
     plugins.set(listOf("yaml"))
 }
 tasks.getByName<org.jetbrains.intellij.tasks.PatchPluginXmlTask>("patchPluginXml") {
-    sinceBuild.set("223")
-    untilBuild.set("223.*")
+    sinceBuild.set("231")
```
Comment (Pakisan) [inline build.gradle.kts:26]: If you replace `223` with `231`, users on 2022.3.* will not receive any new updates.

Did you find some compatibility problems?

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1-rc-1-bin.zip
```
Comment (Pakisan) [inline gradle/wrapper/gradle-wrapper.properties:3]: New version of gradle is out. I propose to you to change this RC version to stable one - 8.1.1

**Review by Pakisan — CHANGES_REQUESTED** [review]

**Comment by alvesf2** [comment]
Hello everybody. Do you have any estimation of when this will be in production?
Cheers!

**Review by Pakisan — APPROVED** [review]

**Comment by Pakisan** [comment]
> Hello everybody. Do you have any estimation of when this will be in production?
> 
> Also, image These links take us to the old repository (https://github.com/Pakisan/jasyncapi-idea-plugin), not this one. It would be nice to update this.
> 
> Apologies for commenting on the PR, but seems like there isn't an issue open to track this work.
> 
> Cheers! 🙇

Hi @alvesf2 

I Will make the proposed changes, by myself, run check compatibility tests, and send them to review today evening(02.05)

**Comment by alvesf2** [comment]
> > Hello everybody. Do you have any estimation of when this will be in production?
> > Also, image These links take us to the old repository (https://github.com/Pakisan/jasyncapi-idea-plugin), not this one. It would be nice to update this.
> > Apologies for commenting on the PR, but seems like there isn't an issue open to track this work.
> > Cheers! 🙇
> 
> Hi @alvesf2
> 
> I Will make the proposed changes, by myself, run check compatibility tests, and send them to review today evening(02.05)

Hi, again @Pakisan Usually how long does it take for IntelliJ to validate this change?

Thank you,
Fábio

**Comment by Pakisan** [comment]
> > > Hello everybody. Do you have any estimation of when this will be in production?
> > > Also, image These links take us to the old repository (https://github.com/Pakisan/jasyncapi-idea-plugin), not this one. It would be nice to update this.
> > > Apologies for commenting on the PR, but seems like there isn't an issue open to track this work.
> > > Cheers! 🙇
> > 
> > 
> > Hi @alvesf2
> > I Will make the proposed changes, by myself, run check compatibility tests, and send them to review today evening(02.05)
> 
> Hi, again @Pakisan Usually how long does it take for IntelliJ to validate these changes?
> 
> Thank you, Fábio

Hi!

> The review may take up to two business days (Monday - Friday, CET timezone, excluding official holidays). You will get a response on the Email used to register in the Marketplace. If you have any questions or feedback, please reach out to the Support team at

I missed with release channel. Re-targeted release to stable now.

You can download the plugin and install it manually from the release page. I pushed all changes to the release branch and attached `.jar`, `.zip`

**Comment by Pakisan** [comment]
> > > Hello everybody. Do you have any estimation of when this will be in production?
> > > Also, image These links take us to the old repository (https://github.com/Pakisan/jasyncapi-idea-plugin), not this one. It would be nice to update this.
> > > Apologies for commenting on the PR, but seems like there isn't an issue open to track this work.
> > > Cheers! 🙇
> > 
> > 
> > Hi @alvesf2
> > I Will make the proposed changes, by myself, run check compatibility tests, and send them to review today evening(02.05)
> 
> Hi, again @Pakisan Usually how long does it take for IntelliJ to validate these changes?
> 
> Thank you, Fábio

Release was approved and available now

**Pull request merged**